code/game/objects/items/weapons/storage/boxes_vr.dm

```diff
@@ -35,4 +35,8 @@
 	starts_with = list(/obj/item/weapon/reagent_containers/food/snacks/canned/brainzsnax = 6)
 
 /obj/item/weapon/storage/box/brainzsnax/red
-	starts_with = list(/obj/item/weapon/reagent_containers/food/snacks/canned/brainzsnax/red = 6)
\ No newline at end of file
+	starts_with = list(/obj/item/weapon/reagent_containers/food/snacks/canned/brainzsnax/red = 6)
+
+/obj/item/weapon/storage/box/freezer
+	storage_cost = ITEMSIZE_COST_NORMAL + 2 //1.5X the size of a medkit. You can carry organs in it!
+	can_hold = list(/obj/item/organ, /obj/item/weapon/reagent_containers/blood, /obj/item/weapon/reagent_containers/glass, /obj/item/weapon/reagent_containers/food/drinks, /obj/item/weapon/reagent_containers/food)
```

Comment:
/obj/item/weapon/reagent_containers/food implicitly includes /obj/item/weapon/reagent_containers/food/drinks. Why double definition?

Reply:
Completely missed that, let me go ahead and fix it.